mobile/src/screens/notification-settings/libs/constants.ts

```diff
@@ -0,0 +1,5 @@
+const NOTIFICATION_SETTINGS_DEFAULT_VALUE = {
+  notification: true,
+};
```

Comment:
```suggestion
} as const;
```
always add `as const` for  constants